### PR TITLE
Include static resources in API bundle

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Changes grunt build to generate all static resources in `build/static` instead of as attachments of the main ddoc, and copy these static resources to `api/build/static` once the build is finished.
Resources in `api/build/static` are separated in `webapp`, `login` and `admin`. 
Changes grunt build to save all default docs in `build/default-docs` instead of as attachments of the main ddoc, and copy these static resources to `api/build/default-docs` once finished.
Moves translations files to `api/resources/translations`. Updates api translation module to get translations from fs instead of ddocs and to query the database with `_all_docs` instead of a view. 

Removes `resource-extraction` service. 

Dev rebuilding / sw generating is unchanged. Build process and grunt tasks are further improved in https://github.com/medic/cht-core/pull/7416

medic/cht-core#7397

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Please have a look, @garethbowen, thanks a lot for your time.

File: api/src/environment.js [hunk 1]
```diff
@@ -48,16 +48,9 @@ module.exports.setDeployInfo = newDeployInfo => {
...
-  }
-
-  return path.resolve(destination);
-};
-
+module.exports.getBuildPath = () => path.join(__dirname, '..', 'build');
+module.exports.getStaticPath = () => path.join(module.exports.getBuildPath(), 'static');
+module.exports.getWebappPath = () => path.join(module.exports.getStaticPath(), 'webapp');
+module.exports.getDefaultDocsPath = () => path.join(module.exports.getBuildPath(), 'default-docs');
+module.exports.getResourcesPath = () => path.join(__dirname, '..', 'resources');
```
Comment (garethbowen) [inline api/src/environment.js:55]: Can these be exported constants instead of functions?

**Review by garethbowen — COMMENTED** [review]
Awesome work! One small change requested inline.

At some point I think we want to get away from copying static files that don't have a compilation step (images, fonts, etc). Now that they don't have to be attached to a ddoc I think we can just load them from the source directory to simplify the build even more. I see you've made more progress in a draft PR, so maybe it's already covered? Either way, don't let it block this PR unless you think it's worth doing at the same time.

File: api/src/environment.js (hunk 1 above)
Comment (dianabarsan) [reply api/src/environment.js]: Done!

**Comment by dianabarsan** [comment]
>  I think we can just load them from the source directory to simplify the build even more
Yea, totally! I'll try to clean things up the next PRs.

**Review by garethbowen — APPROVED** [review]
Awesome, thanks!

**Pull request merged**